doc/installing/development-py35.md

```diff
@@ -0,0 +1,192 @@
+# Installing the OpenQuake Engine for development (Python 3.5)
+
+To develop with the OpenQuake Engine and Hazardlib an installation from sources must be performed.
+
+*The source installation will conflict with the package installation, so you
+must remove the openquake package if it was already installed.*
+
```

Comment:
Notice that below there are instructions about how to make multiple installations coexist (i.e. use a different port for the DbServer) so it is actually possible to have both an installation from packages and one from sources if a virtualenv is used. The sentence before is probably a remnant of the past.